modules/wmco-prerequisites.adoc

```diff
@@ -60,8 +62,9 @@ Hybrid networking with OVN-Kubernetes is the only supported networking configura
 |Supported Windows Server version
 
 |Default VXLAN port
-|Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
+|Windows Server 2019
```

Comment:
I think we should specify `Windows Server 2019 (LTSC)` since the other Windows Server 2019 stream (SAC) is not compatible/supported. Applies in above platform table as well for AWS and Azure

Reply:
> I think we should specify `Windows Server 2019 (LTSC)` since the other Windows Server 2019 stream (SAC) is not compatible/supported. Applies in above platform table as well for AWS and Azure

@saifshaikh48 Where do you see a 2019 SAC stream? https://docs.microsoft.com/en-us/windows/release-health/windows-server-release-info

Reply:
@mtnbikenc @saifshaikh48 Should we leave the LTSC in place, just to be safe and specific?

Reply:
My preference is to just go by what is listed in the `Windows Server release` column of the Windows Server release information [page](https://docs.microsoft.com/en-us/windows/release-health/windows-server-release-info).  If we want to make it clear, I suggest adding the version text in that column.

`Windows Server 2019`
would be
`Windows Server 2019 (version 1809)`

Reply:
That's clear, even to me. @saifshaikh48 ?